android/src/main/kotlin/top/kikt/imagescanner/core/utils/DBUtils.kt

```diff
@@ -519,7 +519,12 @@ object DBUtils : IDBUtils {
             inputStream.close()
         } else {
             val tmpPath = assetEntity?.path!!
-            val tmpFile = File(tmpPath)
+            var tmpFile = File(tmpPath)
```

Comment:
This fix didn't apply to the `saveVideo` actually, it applied to the `saveImage`.

Reply:
sorry😓 ,I just pasted the code to the wrong place

Reply:
I just test the `saveImageWithPath` method ,it has the same problem , before apply the code it returns null
![image](http://p0.qhimg.com/t010b42aa9d9f116615.jpg)
after apply the code it works ok
![image](http://p0.qhimg.com/t010075f0027bca57c2.jpg)

Reply:
So it should be applied to every save methods. Thanks for these further tests.